scripts/arch-chroot-lite

```diff
@@ -142,4 +142,9 @@ chroot_setup "$chrootdir" || die "failed to setup chroot %s" "$chrootdir"
 # arch-chroot-lite already has /etc/resolv.conf managed by the builder
 # scripts, so no need to bind to the host system's resolv.conf here
 
-SHELL=/bin/sh unshare --fork --pid chroot "$chrootdir" "$@"
+# SIGINT/SIGTERM handler allows aborting the command
+set -m
+SHELL=/bin/sh unshare --kill-child --pid chroot "$chrootdir" "$@" &
+trap "kill -9 -- \"$!\"" SIGINT SIGTERM
```

Comment:
`kill -9` doesn't give a chance for `unshare` to kill its child. Since `--kill-child` defaults to SIGKILL, I think it will be enough to use `"kill -- \"$!\""` (send SIGTERM to unshare)